tests/driftbcs2d/dirichlet.usr

```diff
@@ -0,0 +1,325 @@
+c-----------------------------------------------------------------------
+c
+c  USER SPECIFIED ROUTINES:

```

Comment:
It would be good to remove these generic (and out of date) comments with an explanation of what this test case is doing.

Reply:
OK, will do that later.